tests/crypto/zkp-authentication.test.js

```
@@ -0,0 +1,558 @@
+            return {
+              status: 401,
+              json: async () => ({ error: 'Invalid credentials' })
+            };
+          }
+        }
+
+        // Protected resource
+        if (path === '/api/protected-resource') {
+          const authHeader = options.headers?.Authorization;
+
+          if (authHeader && authHeader.startsWith('Bearer ')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)